### PR TITLE
Feature/private events

### PR DESCRIPTION
**Pull request by sebasfavaron**
## Summary

Add support for private events in poap.gallery. This means hiding them in the main page as well as only showing them in the activity page if they are finished

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Testing

Try finding a private event on main page or ongoing private event on activity

## Ticket

https://poap-devs.atlassian.net/browse/PCD-253?atlOrigin=eyJpIjoiNDkwYzAwYzMxNWZkNDhkYWI1NTdhNDM4NDAxNjA4OTciLCJwIjoiaiJ9

## Dependencies

Depends on the server addition of private events (not yet approved):
https://github.com/poap-xyz/poap-server/pull/171

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I didn't commit unnecessary logs

### CODE REVIEW

File: src/store/mutations.js [hunk 1]
```diff
@@ -52,58 +52,55 @@ export async function getIndexPageData() {
...
-      mostRecent: mr,
-      mostClaimed: mC,
-      upcoming: up,
-      highestPoapPower: hPP,
+    if(!mC || ev.tokenCount > mC.tokenCount) {
+      mC = ev
     }
+  }
+
+  if (mr) mr.heading = "Most Recent"
```
Comment (rlajous) [inline src/store/mutations.js:93]: insted of `mr `use `mostResent`

File: src/store/mutations.js [hunk 2]
```diff
@@ -52,58 +52,55 @@ export async function getIndexPageData() {
...
+    ev.transferCount = 0
+    for (let j = 0; j < graphEvents.length; j++) {
+      const gev = graphEvents[j];
+      if(ev.id === parseInt(gev.id)) {
+        ev.tokenCount += parseInt(gev.tokenCount)
+        ev.transferCount += parseInt(gev.transferCount)
       }
+    }
+    let now = new Date().getTime()
+    let evDate = new Date(ev.start_date.replace(/-/g, ' ')).getTime()
```
Comment (rlajous) [inline src/store/mutations.js:71]: `evDate` is event date?

**Review by rlajous — CHANGES_REQUESTED** [review]

File: src/store/mutations.js (hunk 2 above)
Comment (sebasfavaron) [reply src/store/mutations.js:71]: yes, I was keeping consistent with old naming (ev is an event variable)

File: src/store/mutations.js [hunk 3]
```diff
@@ -52,58 +52,55 @@ export async function getIndexPageData() {
...
+  if (mr) mr.heading = "Most Recent"
+  if (up) up.heading = "Upcoming Event"
+  if (mC) mC.heading = "Most Claimed Token"
+
+  return {
+    poapEvents: poapEvents,
+    mostRecent: mr,
+    mostClaimed: mC,
+    upcoming: up,
+    highestPoapPower: hPP, // TODO: discard if no longer used
```
[the remark this answers is not in the document]
Comment (rlajous) [reply src/store/mutations.js]: we can ask tomi and agus but if so, beter resolve now

**Review by rlajous — APPROVED** [review]

**Pull request merged**